packages/UI/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,45 @@
+import { tv } from 'tailwind-variants';
+import Checkmark from './../../assets/icons/Checkmark';
+import type { CheckboxProps } from '.';
+
+export default function Checkbox({
+  checked = false,
+  className,
+  state,
+  ...props
+}: CheckboxProps) {
+  const checkbox = tv({
```

Comment:
tailwind variants (tv) declaration should always be outside the jsx function.
having it inside means it'll re-declare whenever the component re-renders which is going to reduce the performance.